### PR TITLE
Added support for --without-syschar in XCCDF modules, now it is

### PR DESCRIPTION
**Pull request by rsprudencio**
possible to generate OVAL and/or XCCDF results with or without
OVAL System Characteristics.

### CODE REVIEW

**Comment by mpreisler** [comment]
Could you also add this to the manpage?

**Comment by rsprudencio** [comment]
Sorry, forgot about that!

File: utils/oscap.8 [hunk 1]
```diff
@@ -100,6 +100,11 @@ Write XCCDF results into FILE.
 Writes results to a given FILE in Asset Reporting Format. It is recommended to use this option instead of --results when dealing with datastreams.
 .RE
 .TP
+\fB\-\-without-syschar\fR
+.RS
+Don't provide system characteristics in result file.
```
Comment (mpreisler) [inline utils/oscap.8]: in OVAL result file, right? Does this have any effect if I don't request OVAL results or ARF but just request XCCDF results?

Comment (rsprudencio) [inline utils/oscap.8]: Oops, indeed it was kinda misleading, added a more descriptive text for this option.

**Comment by rsprudencio** [comment]
Another unrelated failure in process58 probe, @openscap-jenkins please retest this

**Comment by rsprudencio** [comment]
@openscap-jenkins retest this please

**Pull request merged**